nnstreamer_example/custom_example_LSTM/dummy_LSTM.c

```diff
@@ -70,6 +70,22 @@ pt_exit (void *private_data, const GstTensorFilterProperties * prop)
   free (data);
 }
 
+/**
+ * @brief copy tensor info
+ */
+#define copy_tensor_info(_info, _data) \
+  do { \
+    unsigned int i = 0; \
+    unsigned int num_tensors = 0; \
+    assert (_info != NULL); \
+    assert (_data != NULL); \
+    num_tensors = _info->num_tensors; \
+    assert (0 < num_tensors && num_tensors <= NNS_TENSOR_SIZE_LIMIT); \
+    for (i = 0; i < num_tensors; ++i) { \
+      _info->info[i] = _data->info[i]; \
+    } \
+  } while (0)
+
```

Comment:
```memcpy(_info->info, _data->info, sizeof(_info->info[0]) * _info->num_tensors)``` ?

Reply:
I think... If info has a name, a fixed-size is not guaranteed.

Reply:
_info->info[i] = _data->info[i] will have the same issue for ```char *name```.  _info->info[i].name will be pointing to the same address.

Reply:
If "name" matters for the users, you need g_strdup and g_free for cloned tensors (or reset all names to NULL if that doesn't matter)

Reply:
@myungjoo Thank you for review. updated patch.

Reply:
someone who use this macro must g_free(info->info[i].name) after this patch is merged; but that is out of scope from this file.

Reply:
This example did not have dependency to glib, gstreamer and nnstreamer. 
IMO, we dont need to consider the tensor name here.

If it is necessary to handle this in this example, I recommend to use gst_tensors_info_copy() here, include nnstreamer_plugin_api.h with dependency nnstreamer.